app/src/main/java/org/sopt/dosopttemplate/BirthdayViewHolder.kt

```diff
@@ -0,0 +1,14 @@
+package org.sopt.dosopttemplate
+
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.dosopttemplate.databinding.ItemBirthdayBinding
+
+class BirthdayViewHolder(private val binding: ItemBirthdayBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+
+    fun onBind(friendData: Friend) {
+        binding.ivProfilePicture.setImageResource(friendData.profileImage)
+        binding.tvName.text = friendData.name
+        binding.tvSelfDescription.text = friendData.self_description
```

Comment:
왠지 묶어보고 싶지 않나요?

Reply:
아앗 그런가요...?! 묶..묶겠습니다!!

Reply:
```suggestion
with(binding){
         ivProfilePicture.setImageResource(friendData.profileImage)
         tvName.text = friendData.name
         tvSelfDescription.text = friendData.self_description
}
      
```

with 스코프 함수를 통해 요렇게 묶을 수 있슴당!!

Reply:
와아 짱이네요